target/linux/ath79/dts/qca9557_dongwon_dw02-412h-64m.dts

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9557_dongwon_dw02-412h.dtsi"
+
+/ {
+	model = "Dongwon T&I DW02-412H (64M)";
+	compatible = "dongwon,dw02-412h-64m", "qca,qca9557";
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
```

Comment:
FYI you can reduce duplicate code further by adding labels for partitions.
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7628an_jotale_js76x8.dtsi#L89
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7628an_jotale_js76x8-16m.dts#L10
But that will make the code more hard to read, so feel free to ignore this.

Reply:
I think in this case partitions are few enough that having them twice should be bearable.